osmaxx/api_client/API_client.py

```diff
@@ -92,9 +92,23 @@ def _authorization_headers(self):
 
 
 class LazyChunkedRemoteFile:
-    def __init__(self, *args, download_fu=requests.get, **kwargs):
-        assert kwargs.get('stream', True), "can't chunk without streaming"
-        response = download_fu(*args, stream=True, **kwargs)
+    """Wrapper around to-be-downloaded files
+
+    Can be passed to django.core.files.storage.Storage#save as ``content``
+    if that storage's save logic consumes chunks like ``FileSystemStorage`` does
+    at https://github.com/django/django/blob/1.9.4/django/core/files/storage.py#L252
+
+    Notes:
+        - This is not a file-like object: It has no ``read`` method.
+        - Object initialization opens the connection for downloading, but will not fetch the content until the chunks
+          are requested. If too much time has passed between those steps, the server might have closed the connection,
+          so don't wait too long.
+    """
+
+    def __init__(self, *args, download_function=requests.get, **kwargs):
+        stream = kwargs.pop('stream', True)
+        assert stream, "can't chunk without streaming"
```

Comment:
Just checking if this is the desired behavior: This will only raise if stream is explicitly set to False, if it isn't set, it will set it to true and work.